libkbfs/block_retrieval_queue.go

```diff
@@ -66,6 +69,14 @@ type blockPtrLookup struct {
 	t  reflect.Type
 }
 
+// blockRetriever specifies a method for retrieving blocks asynchronously
+type blockRetriever interface {
```

Comment:
Huh?  I was just talking about moving the definition of the interface, not changing anything about it.  It just doesn't seem like it belongs in this file.